### PR TITLE
feat: add vim-dadbod-ui integration

### PR DESCRIPTION
**Pull request by vihu**
Summary
----
This adds support for [dadbod-ui](https://github.com/kristijanhusak/vim-dadbod-ui).

I chose the following colors (mostly because it seemed reasonable):

```
NotificationInfo = { fg = C.blue, bg = C.mantle },
NotificationWarning = { fg = C.yellow, bg = C.mantle },
NotificationError = { fg = C.red, bg = C.mantle },
```

PS: There's some whitespace cleanup here as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -477,6 +477,20 @@ colorful_winsep = {
...
 <!-- colorful_winsep.nvim -->
 
+<!-- dadbod-ui -->
+</tr>
+<tr>
+<td> <a href="https://github.com/kristijanhusak/vim-dadbod-ui">dadbod-ui</a> </td>
+<td>
+
+```lua
+dadbod_ui = true
```
Comment (vollowx) [inline README.md:487]: I guess `false` actually, as well as the vimdoc below.

File: doc/catppuccin.txt [hunk 2]
```diff
@@ -118,7 +118,7 @@ options and settings.
             -- For more plugins integrations please scroll down (https://github.com/catppuccin/nvim#integrations)
         },
     })
-
```
Comment (vollowx) [inline doc/catppuccin.txt:121]: They're automatically generated by the vimdoc bot right? I guess these whitespaces will be back some time in the future.

**Review by vollowx — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vihu) [reply README.md]: Updated in [5b382d4](https://github.com/catppuccin/nvim/pull/747/commits/5b382d402dd30ad0a2f94400853e22a04595e1a1)

File: README.md [hunk 3]
```diff
@@ -477,6 +477,20 @@ colorful_winsep = {
 </tr>
 <!-- colorful_winsep.nvim -->
 
+<!-- dadbod-ui -->
```
Comment (vollowx) [inline README.md:480]: Change to `vim-dadbob-ui` to show its real name, as well as line 483 and 492, but no need to change the integration name in line 487. Also remember to move this chunk down to integrations start with `v`.

**Review by vollowx — APPROVED** [review]

**Comment by vollowx** [comment]
Thanks for your contribution and patience! :heart:

**Pull request merged**